cmake/Util.cmake

```diff
@@ -0,0 +1,58 @@
+# Defines a target that depends on ${FILES} and the files found by globbing
+# when using ${GLOB_PAT} and ${GLOB_DIRS}. The target will only run if any of
+# the files it depends on is changed.
+#
+# REQUIRED  - Abort if COMMAND doesn't exist.
```

Comment:
This function doesn't have any business doing find_foo(). Just pass a program path to it, maybe with a `fatal` flag like the old `def_cmd_target`. 

Also if it took a list instead of a glob (perhaps optionally?) then we could use this for "lintc".

Reply:
It does take a list. SMART_FILES is a list. The glob is optional.

Reply:
Ye, I'll remove the find code